lib/ae_page_objects/elements/collection.rb

```diff
@@ -66,10 +73,9 @@ def item_class_at(index)
 
     def item_xpath
       @item_xpath ||= begin
-        evaled_locator = eval_locator(@item_locator)
-        
-        query_args = evaled_locator + [{:exact => true}]
```

Comment:
I have created a separate PR for the `{:exact => true}` change.  It turned out it was needed but was not obvious from the code.  See https://github.com/appfolio/ae_page_objects/pull/190 for details.